src/ceph-detect-init/tests/test_all.py

```diff
@@ -56,6 +60,10 @@ def test_debian(self):
                                  distro='ubuntu',
                                  codename='vivid'):
             self.assertEqual('systemd', debian.choose_init())
+        with mock.patch.multiple('ceph_detect_init.debian',
+                                 distro='not-debian',
```

Comment:
it won't match, i think. but "Deb" will. because `'non-debian'.lower() in ('debian')` is `False`.

yeah, i will go with 
```py
if distro.lower() == 'debian':
```